cogs/make_member.py

```
@@ -123,159 +123,244 @@ async def make_member(self, ctx: "TeXBotApplicationContext", group_member_id: st
+                await interaction_member.remove_roles(
+                    applicant_role,
+                    reason='TeX Bot slash-command: "/makemember"',
                 )
-            )
 
-        guild_member_ids.discard("")
-        guild_member_ids.discard("\n")
-        guild_member_ids.discard(" ")
-
-        if not guild_member_ids:
-            await self.command_send_error(
-                ctx,
-                error_code="E1041",
-                logging_message=OSError(
-                    "The guild member IDs could not be retrieved from "
-                    "the MEMBERS_LIST_URL.",
-                ),
+class MemberCountCommandCog(TeXBotBaseCog):
+    """Cog class that defines the "/membercount" command and its call-back method."""
+
+    @discord.slash_command(  # type: ignore[no-untyped-call, misc]
+        name="membercount",
+        description="Displays the number of members in the group.",
+    )
+    async def member_count(self, ctx: "TeXBotApplicationContext") -> None:
+        """Definition & callback response of the "member_count" command."""
+        await ctx.defer(ephemeral=True)
+        async with ctx.typing():
+            http_session: aiohttp.ClientSession = aiohttp.ClientSession(
+                headers=REQUEST_HEADERS,
+                cookies=REQUEST_COOKIES,
             )
-            return
-
-        if group_member_id not in guild_member_ids:
-            await self.command_send_error(
-                ctx,
-                message=(
-                    f"You must be a member of {self.bot.group_full_name} "
-                    "to use this command.\n"
-                    f"The provided {_GROUP_MEMBER_ID_ARGUMENT_NAME} must match "
-                    f"the {self.bot.group_member_id_type} ID "
-                    f"that you purchased your {self.bot.group_short_name} membership with."
-                ),
+            async with http_session, http_session.get(REQUEST_URL[:-12]) as http_response:
```

Using a slice of the members list URL is not robust/ easily changeable if the Guild website's URL structure changes.
Please choose one of the following alternatives:
* Create 2 separate environment variables for each request URL
* Create one base URL with additional paths for each request
* Use a regex on the existing members list URL that will fail hard if the URL structure is not what is expected

---

Modification of environment variables is out of scope for this PR so will be done separately.